### DISCUSSION TITLE
auth.signUp with emailRedirectTo inside options

### DISCUSSION THREAD

**Discussion by sambeet21** [supabase/supabase#27031] [Questions]
Hi I am facing one issue while I am sending the emailRedirectTo inside the options in auth.signUp
for example 
supabase.auth.signUp({ email: "abc@gmail.com", password: "welcome", options: {emailRedirectTo: "https:site.com/auth/callback"} })

when I receive mail I am not getting the url as emailRedirectTo in email confirmation URL {{.RedirectTo}} 
the value {{RedirectTo}} is the siteURL which we set up in the URL configuration 

please help

**Comment by GaryAustin1** [comment]
This usually  means your redirectTo URL is not an EXACT full match for one in the Additional Redirect URLs settings in the Dashboard.

**Reply by sambeet21** [reply]
Oh okay got it where is that issue may be I am appending the /auth/callback as well and along with the url i need to add auth/callback and whitelist that in that supabase dashboard

**Reply by sambeet21** [reply]
@GaryAustin1 No I tried giving exact one but still not working
